### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by marcelo-franceschini**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/howto/docker-compose/index.rst [hunk 1]
```diff
@@ -160,7 +160,7 @@ Now you can start all services:
 
 .. code-block:: bash
 
-    docker-compose up
+    docker compose up
```
Comment (eladkal) [inline docs/apache-airflow/howto/docker-compose/index.rst:163]: this is not a mistake
see https://github.com/apache/airflow/pull/27792 and https://github.com/apache/airflow/pull/28194

I will raise a PR to clarify that this is not a mistake as this is the 3rd time people are asking to fix it :)

**Comment by eladkal** [comment]
closed in favor of https://github.com/apache/airflow/pull/28729

File: docs/apache-airflow/howto/docker-compose/index.rst (hunk 1 above)
Comment (marcelo-franceschini) [reply docs/apache-airflow/howto/docker-compose/index.rst:163]: Sorry, I just wanted to help. But it is still unclear to me.
As I understood in [#28194](https://github.com/apache/airflow/pull/28194#issuecomment-1341479006), Airflow is using docker compose v2.
image

image

Comment (marcelo-franceschini) [reply docs/apache-airflow/howto/docker-compose/index.rst:163]: Current Page
image

Comment (eladkal) [reply docs/apache-airflow/howto/docker-compose/index.rst:163]: if there are places that we use v1 syntax then you can open PR to update v2 but not the other way around :)
documntation is always improving so it's cool that people like yourself care about improving it 👍

**Pull request closed without merging**